Makefile

```diff
@@ -35,7 +36,7 @@ quality:
 fix:
 	./node_modules/stylelint/bin/stylelint.js sass/**/*.scss samples/**/*.scss --fix
 
-test: clean quality samples ## run tests
+test: clean quality ## run tests
```

Comment:
I don't love losing this target, because now we don't verify that the Sass can compile. Could you maybe have a new target to build but not open the samples?

Reply:
Yep, see #16.

Reply:
Nevermind, we moved to #17.